app/src/main/res/layout/fragment_search.xml

```diff
@@ -20,55 +20,114 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent">
 
+            <RadioGroup
```

Comment:
Think of a way to do it without radio buttons. You don't use any of the radio button functionality so there is no need to use it.